paasta_tools/cli/cmds/logs.py

```diff
@@ -1346,11 +1392,19 @@ def paasta_logs(args: argparse.Namespace) -> int:
     else:
         clusters = args.clusters.split(",")
 
+        if len(clusters) > 1:
+            print("You can only filter one cluster.")
+            return 1
+
```

Comment:
Instead of constraining clusters and instances here, I think you should just change the arg parser to only allow one arg instead of being an narg/multi-arg.

Reply:
I wasn't using nargs for clusters and instances initially, but I've changed it to have that option which makes the arguments one list of one item for both clusters and instances.

Example for multiple clusters referenced:
`(py36-linux) lancelei@dev54-uswest1adevc:~/Repos/paasta$ paasta logs -s mtb_ranking -c norcal-devc,pnw-devc
Fetching 100 lines and applying filters. Try -n 1000 for more lines...
I don't know where scribe logs for norcal-devc,pnw-devc live?`

Example for multiple instances referenced:
`(py36-linux) lancelei@dev54-uswest1adevc:~/Repos/paasta$ paasta logs -s mtb_ranking -c norcal-devc -i long_timeout,main
mtb_ranking doesn't have any instances matching long_timeout,main on norcal-devc.
Did you mean any of these?
  long_timeout`